app/helpers/facility_orders_helper.rb

```diff
@@ -9,18 +9,16 @@ def order_detail_notices(order_detail)
     notices << "can_reconcile" if order_detail.can_reconcile_journaled?
     notices << "in_open_journal" if order_detail.in_open_journal?
 
-    warnings = []
-    warnings << "missing_actuals" if order_detail.complete? && order_detail.reservation.try(:requires_but_missing_actuals?)
-    warnings << "missing_price_policy" if order_detail.missing_price_policy? && !order_detail.reservation.try(:requires_but_missing_actuals?)
+    warnings = [order_detail.problem_description]
 
     { warnings: warnings, notices: notices }
   end
 
   def order_detail_badges(order_detail)
     notices = order_detail_notices(order_detail)
 
-    output = build_badges(notices[:warnings], "label-important")
-    output += build_badges(notices[:notices], "label-info")
+    output = build_warnings(notices[:warnings])
+    output += build_notices(notices[:notices])
```

Comment:
Is this something we can use `safe_join` for and get rid of the extra `html_safe` marker?